client/X11/xf_keyboard.c

```diff
@@ -191,6 +194,28 @@ void xf_keyboard_send_key(xfContext* xfc, BOOL down, BYTE keycode)
 	DWORD rdp_scancode;
 	rdpInput* input;
 	input = xfc->context.input;
+
+	if (keycode == 0x25)
```

Comment:
Better check against `rdp_scancode` and the `RDP_SCANCODE*` macros instead of these magic numbers